lib/datapacktypes/omniscript.js

```diff
@@ -2,6 +2,7 @@ var vlocity = require('../vlocity');
 var puppeteer = require('puppeteer-core');
 var fs = require('fs-extra');
 var path = require('path');
+const chromium = require('chromium');
```

Comment:
We won't bundle chromium as it will increase the size of the app too much.

Reply:
Done.. I have removed chromium and Other small changes @rutlabaga